libyul/AsmPrinter.cpp

```diff
@@ -278,7 +278,10 @@ string AsmPrinter::formatSourceLocation(
 	{
 		sourceIndex = to_string(_nameToSourceIndex.at(*_location.sourceName));
 
-		if (_debugInfoSelection.snippet && _soliditySourceProvider)
+		if (_debugInfoSelection.snippet &&
```

Comment:
```suggestion
        if (
            _debugInfoSelection.snippet &&
```

styling not quite correct

Reply:
(ignore my fucked up space-based indentations, github makes it hard)